src/components/EventsForm.tsx

```diff
@@ -0,0 +1,226 @@
+  title: string;
+  capacity: number;
+  thumbnail?: string;
+  price: number;
+  description: string;
+  location: string;
+  start: Moment;
+  end: Moment;
+}
+
+export interface EventsFormProps {
+  onFinish: (data: EventsFormData) => Promise<void>;
+  tokens: UserAuthenticationReducerState['tokens'];
+  initialValues?: EventsFormInitialValues;
+  edit: boolean;
+}
+
+const EventsForm: React.FC<EventsFormProps> = ({
+  onFinish,
+  tokens,
+  edit,
+  initialValues,
+}) => {
+  return (
+    <FormContainer>
+      <Helmet>
+        <title>Create Event</title>
+      </Helmet>
+      <Form
+        name="basic"
+        layout="vertical"
+        initialValues={initialValues}
+        onFinish={onFinish}
+      >
+        <Form.Item
+          label="Event Name"
+          name="title"
+          rules={[
+            { required: true, message: 'Please input the name of the event' },
+          ]}
+        >
+          <Input placeholder="Event Name" />
+        </Form.Item>
+
+        <Form.Item
+          label="Event Location"
+          name="location"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the location of the event',
+            },
+          ]}
+        >
+          <Input placeholder="Event Location" />
+        </Form.Item>
+
+        <Form.Item
+          name="start"
+          fieldKey="start"
+          style={{ display: 'inline-block' }}
+          label="Start Date"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the start date and time of the event',
+            },
+          ]}
+        >
+          <DatePicker
+            style={{ marginRight: '8px' }}
+            showTime
+            use12Hours
+            format="MM/DD/YYYY hh:mm A"
+          />
+        </Form.Item>
+
+        <p
+          style={{
+            display: 'inline-block',
+            marginLeft: '8px',
+            marginRight: '8px',
+            marginTop: '32px',
+          }}
+        >
+          to
+        </p>
+
+        <Form.Item
+          name="end"
+          fieldKey="end"
+          style={{ display: 'inline-block' }}
+          label="End Date"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the end date and time of the event',
+            },
+          ]}
+        >
+          <DatePicker
+            style={{ marginLeft: '8px' }}
+            showTime
+            use12Hours
+            format="MM/DD/YYYY hh:mm A"
+          />
+        </Form.Item>
+
+        <Form.Item
+          label="Spots Available"
+          name="capacity"
+          rules={[
+            {
+              required: true,
+              message: 'Please input the number of spots available',
+            },
+          ]}
+        >
+          <InputNumber
+            placeholder="Spots Available"
+            min={0}
+            style={{ width: '50%' }}
```

Comment:
🔧 Same here, we should use styled components

Reply:
I dont wanna spam comments so if theres more `style` props below then know that you should also rewrite those too